### ISSUE TITLE
[Accessibility]HighContrast: ToolStripButtons in PropertyGrid should have obvious distinct between focused and checked status

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#4268]
* .NET Core Version: 
6.0.100-aipha.1.20573.16

* Have you experienced this same bug with .NET Framework?: 
Yes

**Minimal repro:**
1. Extract following attached app and open it in VS
[WinFormsApp30.zip](https://github.com/dotnet/winforms/files/5588567/WinFormsApp30.zip)
2. Set Windows as HighContrast modes: HC Black/White/#1/#2
3. Build and run the project.
4. Press "Right/Left" arrow to focus on "Categorized", "Alphabetical" and " Property Pages" toolStripbuttons.

**Actual behavior:**
ToolStripButtons in PropertyGrid have no obvious distinct between focused and checked status in HC themes
![image](https://user-images.githubusercontent.com/26474449/100072081-c5886100-2e76-11eb-92f5-390882b13f65.png)

**Expected behavior:**
Make focused and checked tool strip buttons look more distinct in HC themes
